samples/automl/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "automl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "automlVisionDataset.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Would be great to be able to run the tests directly from here.

Reply:
Fixed in latest commit.